client/src/components/posts/PostDetails.js

```diff
@@ -51,7 +53,9 @@ export const PostDetails = ({ loggedInUser }) => {
     }
 
     useEffect(() => {
-        getPostById();
+        fetchReactions().then(setReactions).then(() => {
```

Comment:
Is there a reason that the reactions have to be fetched before the post? It seems like they could both be fetched at the same time, then you can set state when each comes back.

Reply:
it's because I was getting an error when the post was set first because the 'if (!post) return' safety net didn't protect from reactions not being set, and the reactions state only needs to be set once

Reply:
I was going to do that, just seemed like a potential merge conflict for people so I just went ahead and got reactions once before setting the post

Reply:
ok